common/src/main/java/com/microsoft/identity/common/internal/controllers/BaseController.java

```diff
@@ -77,6 +75,10 @@
 
     private static final String TAG = BaseController.class.getSimpleName();
 
+    private static final String OPEN_ID_SCOPE = "openid";
```

Comment:
nit: I suggest putting these values in AuthenticatorConstants, as these values is also being used somewhere else (PRTController)

Reply:
Thanks for the suggestion @rapong . Addressed it